app/views/study_sessions/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1> Topic: <%= @studySession.topic %> </h1>
+<h2 id="counter"></h2>
+<h2> Paired: <%= @studySession.paired %></h2>
+<body>
+  <script type="text/javascript">
+    function countdown(minutes) {
+    var seconds = 60;
+    var mins = minutes
+    function tick() {
+        var counter = document.getElementById("counter");
+        var current_minutes = mins-1
+        seconds--;
```

Comment:
SO what is this line doing? Pardon my JS ignorance. I see on line 11 the `mins-1`. Is this reducing the minute by 1 and then the seconds timer on line 13 ticks down?